specification/sql/clear_cursor/ClearSqlCursorRequest.ts

```diff
@@ -20,6 +20,7 @@
 import { RequestBase } from '@_types/Base'
 
 /**
+ * Clear an SQL search cursor.
```

Comment:
Do we have any style guidance for which article to use -- "an SQL" or "a SQL" ? 

It looks like "a SQL" has a slight edge in the es docs, and the [Google dev docs guide](https://developers.google.com/style/word-list#sql) prefers "a" -- but there might be a precedent I don't know about. I can see arguments on both sides of this one!

Reply:
Looks like our current (internal) style guide recommends "an SQL", which I didn't know until you mentioned this.. but good to know it's covered!